Docker/ods-api-db-ods-minimal/ubuntu/mssql/Dockerfile

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+# Base image with additional packages
+FROM mcr.microsoft.com/mssql/server:2022-CU13-ubuntu-22.04@sha256:c4369c38385eba011c10906dc8892425831275bb035d5ce69656da8e29de50d8 AS base
+USER root
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get -y install --no-install-recommends unzip=6.0-26ubuntu3 dos2unix=7.4.2-2 &&\
+    rm -rf /var/lib/apt/lists/* &&\
+    # These steps are needed in order to mount the database to a host volume
+    mkdir -p /var/opt/mssql/data &&\
+    chown -R mssql: /var/opt/mssql/data
+USER mssql
+
+# Start a new layer so that the above layer can be cached
+FROM base AS build
+ARG ODS_VERSION
+ARG TPDM_VERSION
```

Comment:
Should we rename these to `ODS_MINIMAL_VERSION` and `TPDM_MINIMAL_VERSION` to be consistent with the other new images? (Note that `docker-compose-local-mssql.yml` is passing the wrong args as of now).

Reply:
We can do this for clarity, but also will need to replicate the change in the pgsql image